src/utils/api/emoji.ts

```diff
@@ -0,0 +1,60 @@
+import { emojiController } from "../libs/requestUrls";
+import RequestApiV2 from "../libs/requestApi";
+import { EmojiType } from "types/types";
+import { AxiosResponse } from "axios";
+import { emojiRes } from "types/api";
+
+class Emoji {
+  getEmoji(emoji: number): Promise<void | AxiosResponse<emojiRes>> {
+    try {
+      return RequestApiV2({
+        url: emojiController.getEmogi(emoji),
+        method: "GET",
+      });
+    } catch (e: any) {
+      throw new Error(e);
+    }
+  }
+
+  addEmoji(
```

Comment:
함수의 인자값은 1~2개가 이상적입니다.
그 이상일 경우, 대부분은 함수가 하는 일이 많다는 것의 반증이기도 하죠

값을 넘기기위한 필요값들이라면 container, dto 같은 인터페이스들을 이용하여
비구조화문법을 사용하여 객체를 주고받고 하는것이 좋지않을까 싶습니다.
```suggestion
  addEmoji(
  { isLogin, emoji, number }
    : {
    isLogin: boolean,
    emoji: EmojiType,
    number: number
  }
```

Reply:
공부를 해서 배워서 빠른 시일내에 수정하겠습니다.